src/tiled/tilesetdock.cpp

```diff
@@ -28,6 +28,7 @@
 #include "containerhelpers.h"
 #include "documentmanager.h"
 #include "erasetiles.h"
+#include "fileformat.h"
```

Comment:
Already includes `tmxmapformat.h` and `tilesetformat.h`, so this is superfluous.

Reply:
Done

I was having forward declare compile issue so I had tried to include directly to mitigate. Wasn't necessary and I hadn't reverted, good catch.